action.yml

```diff
@@ -94,9 +104,19 @@ runs:
         echo "Either the 'appSourcePath' or 'imageToDeploy' argument must be provided when providing the 'acrName' argument."
         exit 1
 
-    - name: Install pack CLI
+    - name: Install pack CLI on non-Windows runner
+      if: ${{ runner.os != 'Windows' }}
       shell: bash
-      run: (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
+      run: (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-${{ env.RUNNER_OS }}.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
+
+    - name: Install pack CLI on Windows runner
+      if: ${{ runner.os == 'Windows' }}
+      shell: bash
+      run: |
+          mkdir -p $PWD/pack && cd $PWD/pack
+          curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.27.0/pack-v0.27.0-windows.zip" -o "pack-windows.zip"
+          7z x pack-windows.zip > /dev/null 2>&1
```

Comment:
[Nothing to change] For those of you that might have the same question as I did: it looks like 7z is installed by default in runners using Windows.

Reply:
Good callout! [Here is the list of preinstalled software for each provided (_i.e._, not managed by the user) GitHub Action runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#preinstalled-software)